tdameros/42-transcendence
# Ground work for pong renderer and server

Made a server (redirection server) that redirects users to another server (game server)

Made a ThreeJS engine capable of:
- connecting to the redirection server and game server while rendering a loading screen
- receiving scenes from the game server (this is only implemented on the client side for now)



To test the redirection part:
1. Create .venv in PROJECT_ROOT/pong_server using PROJECT_ROOT/pong_server/docker/requirements.txt
3. run `cd PROJECT_ROOT/pong_server && source .venv/bin/activate && ./rerun.sh` (if it fails: run `brew install entr`)

4. go to PROJECT_ROOT/front/src/threejs
5. run `npm i`
6. run `npx vite`
7. open the link shown by `npx vite` in a browser

Open the browser console to see the logs
